### ISSUE TITLE
harbor_config_auth.ldap_verify_cert seem to fail to apply

### ISSUE THREAD

**Issue by EugenMayer** [goharbor/terraform-provider-harbor#294]
Either it fails to apply or it fails to fetch the state after applying it. It re-applys it on every run. 

Latest provider (`3.7.1`) and latest tf (`1.3.8`)

**Comment by flbla** [comment]
Hello,
It should have been fixed with the 3.8.0.
You are probably still using the "BESTSELLER" provider.
Could you please try with registry.terraform.io/goharbor/harbor ?

**Comment by EugenMayer** [comment]
oh, did not notice the migration. Sorry!

Switched to  
```
 harbor = {
      # @see https://registry.terraform.io/providers/goharbor/harbor/latest
      source = "goharbor/harbor"
      version = "3.8.2"
    }
```

And the issue is gone. Please feel free to close the issue anytime! Thanks!
